### ISSUE TITLE
Grade report alignment horizontal scroll on small screen size

### ISSUE THREAD

**Issue by mnods** [gjb2048/moodle-theme_essential#879]
using 3.5.1.0 and 3.4.1.2 

i recall this being an issue in an earlier version. 

on smaller screen size in grader report, (grade/report/grader/index.php) using horiz scroll with a fair number of students, the column that sticks (div with classes "floater sideonly floating") is vertically misaligned, making it very hard to grade properly, especially with a large class.

altering the inline css to
top: 88px;

seems to fix it.

seems ok in clean.

**Comment by gjb2048** [comment]
Ok 'altering the inline css to top: 88px;' won't work because that value is generated (and changed) by the underlying JavaScript.  The issue is caused by the core JavaScript being written for the specifics of the themes in core Moodle and thus is not adaptable / flexible enough to cope with changes in contributed themes - its written in a constraining way.  The solution is to disable that JS (somehow) and use theme specific JS to implement the same functionality or find another solution to the issue of not seeing the table headers when there are a large number of columns.  This is not 'trivial' as I think the markup output is difficult to override.

**Comment by mnods** [comment]
yes gareth, i get that i can't change the inline generated css, just wondered if there's a reasonably simple adjustment that can be made to the css for below 979px that is causing the extra height for that floating column calculation. the height calc looks fixed to me. not easy i can see. weird that a refresh resets it.

**Comment by gjb2048** [comment]
I've previously spent 'days' on this!  I'll look again when I get a moment.  This is not easy!!

**Comment by gjb2048** [comment]
Possibly.

**Comment by gjb2048** [comment]
This one is difficult, so going to leave for now.
